addons/binding/pom.xml

```diff
@@ -113,6 +113,7 @@
     <module>org.openhab.binding.mqtt.generic.test</module>
     <module>org.openhab.binding.mqtt.test</module>
     <module>org.openhab.binding.nadreceiver</module>
+    <module>org.openhab.binding.nanoleaf</module>
```

Comment:
?

Reply:
Nanoleaf is another binding - From my side I "just" implemented "nadreceiver" binding and made a PR. Don't know why it was in conflict with this other binding.

Reply:
@davidgraeff your looking at the merge commit because @marcdevinci didn't push force. The change you refer to won't show up if you look at the 'Files changed' tab.